docs/src/main/sphinx/installation/cli.rst

```diff
@@ -287,3 +292,129 @@ To view debug information, including the stack trace for failures, use the
+      immediately).
+  * - ``--insecure``
+    - Skip validation of HTTP server certificates (should only be used for
+      debugging).
+  * - ``--keystore-path``
+    - The location of the Java Keystore file that is used to secure TLS. Connect
+      to a cluster that uses :ref:`cli-certificate-auth`.
+  * - ``--keystore-password``
+    - The password for the keystore. This must match the password you specified
+      when creating the keystore. Connect to a cluster that uses
+      :ref:`cli-certificate-auth`.
+  * - ``--keystore-type``
+    - Keystore type
+  * - ``--krb5-config-path``
+    - Kerberos configuration files.
+  * - ``--krb5-credential-cache-path``
+    - Kerberos credential cache path.
+  * - ``--krb5-disable-remote-service-hostname-canonicalization``
+    - Disable service hostname canonicalization using the DNS reverse lookup.
+  * - ``--krb5-keytab-path``
+    - The location of the keytab that can be used to authenticate the principal
+      specified by ``--krb5-principal``.
+  * - ``--krb5-principal``
+    - The principal to use when authenticating to the coordinator.
+  * - ``--krb5-remote-service-name``
+    - Trino coordinator Kerberos service name.
+  * - ``--krb5-service-principal-pattern``
+    - Remote kerberos service principal pattern (default: ${SERVICE}@${HOST})
+  * - ``--network-logging``
+    - Network logging level [NONE, BASIC, HEADERS, BODY] (default: NONE).
+  * - ``--output-format``
+    - :ref:`cli-output-format` for batch mode [ALIGNED, VERTICAL, TSV,
+      TSV_HEADER, CSV, CSV_HEADER, CSV_UNQUOTED, CSV_HEADER_UNQUOTED, JSON,
+      NULL] (default: CSV).
+  * - ``--password``
+    - Prompt for a password, if your Trino server requires password
+      authentication. For more information, see
+      :ref:`cli-username-password-auth`.
+  * - ``--progress``
+    - Show query progress in batch mode.
+  * - ``--resource-estimate``
+    - Resource estimate (property can be used multiple times; format is
+      key=value).
+  * - ``--schema``
+    - Default schema.
+  * - ``--server``
+    - The address and port of the Trino coordinator. The port must be set to
+      the port the Trino coordinator is listening for HTTPS connections on.
+      Trino server location (default: localhost:8080). The Trino CLI does not
+      support using ``http`` scheme for the URL when using LDAP authentication.
+  * - ``--session``
+    - Session property (property can be used multiple times; format is
+      key=value; use 'SHOW SESSION' to see available properties).
+  * - ``--session-user``
+    - Username to impersonate.
+  * - ``--socks-proxy``
+    - SOCKS proxy to use for server connections.
+  * - ``--source``
+    - Name of source making query (default: trino-cli).
+  * - ``--timezone``
+    - Session time zone (default: America/New_York).
+  * - ``--trace-token``
+    - Trace token.
+  * - ``--truststore-password``
+    - The password for the truststore. This must match the password you
```

Comment:
All these should be mentioned in tls section (and link to it here as well probably)